app/javascript/application.js

```diff
@@ -0,0 +1,3 @@
+// Entry point for the build script in your package.json
+import "@hotwired/turbo-rails"
+import "./controllers"
```

Comment:
- [ ] This is the javascipt folder it should be

Reply:
@Olamarx bro I am not getting what you wrote here. Please try to write in a clear way

Reply:
This is the folder your react code should be, not in the asset folder. Do you understand? We were talking on Slack and you said till tomorrow, I am just getting to see this.